gcpy/benchmark.py

```diff
@@ -3402,6 +3420,13 @@ def make_benchmark_aerosol_tables(
                                    drop_variables=gcon.skip_these_vars)  # ,
         # combine="nested", concat_dim="time")
 
+    # Rename SpeciesConc_ to SpeciesConcVV_ for consistency with new
+    # naming introduced in GEOS-Chem 14.1.0
+    for v in ds_spc.data_vars.keys():
+        if v.startswith('SpeciesConc_'):
+            spc = v.replace('Species_Conc', '')
```

Comment:
Same typo here, should be `SpeciesConc_`

Reply:
Thanks for catching that. The typo is now fixed in [ae12821f](https://github.com/geoschem/gcpy/commit/ae12821fdb30aaf47d90d38c28079159a822f80b).